x-pack/plugins/osquery/public/saved_queries/saved_queries_dropdown.tsx

```diff
@@ -96,12 +95,14 @@ const SavedQueriesDropdownComponent: React.FC<SavedQueriesDropdownProps> = ({
     ({ value }) => (
       <>
         <strong>{value.id}</strong>
-        <TextTruncate>
+        <div css={textTruncateCss}>
```

Comment:
EUI has a CSS utility for this, if you're interested :) 
```suggestion
        <div className="eui-textTruncate">
```